locales/de/common.json

```diff
@@ -1 +1,3 @@
-{}
+{
+  "ImprintAndContact": "Impressum und Kontakt"
```

Comment:
true, i18next-resource-store-loader is used to generate the translations.
The impementation of the translations is in i18n. Different ns (namespaces) mean different files. We could merge all C
component translations in one json file